### DISCUSSION TITLE
How can I train a 48khz or a 44.1khz VITS model?

### DISCUSSION THREAD

**Discussion by marcos33998** [coqui-ai/TTS#3987] [General Q&A]
I'm having problem training at a hight sampling rate, I'm not very good at code but I thought that it is just changing some parameter and ask chatgpt, but I'm faced problems like n_mel is wrong there is a mismatch, tensor a must be the same as tensor b. I couldn't find any help or the way I'm trying is wrong.

And also I have some questions to the VITS training, first thing is that I don't know how to continue a training from a checkpoint, I let chatgpt if the code has some hints but I didn't managed to continue a training, but actually it isn't very important as I feel my computer pretty consistant but sometimes windows shuts down to update...

The output model is a single .pth, I've used VITS models and they were G_xxxxx.pth and a D_xxxxx.pth and both weren't this big. I trained a 22050hz model to try, and I failed to load that model into my program, and maybe it's the config.json's fault as it is very different to others vits models I've tried...

Also I left default the phoneme language: phoneme_language="en-us", as I'm training for a person that speaks en-us I didn't change.
The code i used to train 22050hz is the default and I hope someone teach me how to train at 48khz or 44.1khz :)

**Comment by SMK1311** [comment]
你成功了吗，我也想训练48khz

**Reply by marcos33998** [reply]
not quite, you can check https://www.kaggle.com/datasets/noml4u/tts-models--en--jenny-dioco--vits

**Reply by SMK1311** [reply]
谢谢你的回答，有一个问题困扰我好久，就是我在微调的时候grad_norm总是0，
--> TIME: 2025-05-28 09:50:28 -- STEP: 50/100 -- GLOBAL_STEP: 17650
     | > loss_text_ce: 0.019145922735333443  (0.015993597023189067)
     | > loss_mel_ce: 0.07107582688331604  (0.058668266385793685)
     | > loss: 0.09022174775600433  (0.07466186337172986)
     | > grad_norm: 0  (0.0)
     | > current_lr: 5e-06 
     | > step_time: 0.4572  (0.44223365306854245)
     | > loader_time: 0.0109  (0.008653001785278324)
  您遇见过这种情况吗

**Reply by marcos33998** [reply]
🤷‍♂️, if it works then leave it
